crates/sncast/tests/data/accounts/accounts.json

```diff
@@ -47,10 +47,17 @@
             "public_key": "0x4fdaeb60e75399877bd5d14d3dac2e805f3c2a6732851a19db63fee0cfae00d",
             "address": "0x68d6e92b8e2310c550178b2fe484d5cb89dd6aa93ae804afd4d124e9d6c3cd3"
         },
-        "user9": {
+        "cairo1": {
```

Comment:
these are cairo0 accounts, right?

Reply:
After setting `--account-class cairo1` in the `starknet-devnet` command, all predeployed accounts are Cairo 1.

Reply:
let me rephrase my question - what are the accounts you have added here?

Reply:
I've renamed `user9` -> `cairo1` and included the account that we deploy in the `deploy_keystore_account` fixture to this file (`cairo0`). I haven't added any new account to our devnet network that wasn't already present.

Reply:
> I've renamed user9 -> cairo1

what is the rationale behind this?

> included the account that we deploy in the deploy_keystore_account 

keystore accounts should never be added to accounts file, these are different types altogether and shouldn not be mixed

Reply:
> what is the rationale behind this?

This account is used in parameterized tests, that execute the same command for 2 accounts. It's more self explanatory to have `--account cairo1`, rather that `--account user9`.

> keystore accounts should never be added to accounts file, these are different types altogether and should not be mixed

We don't use this account in any other test, by reusing it there is no need to add redundant code. However, if you prefer to use the keystore account only for deployment, I can generate a separate account for Cairo 0 and add a fixture that deploys it.

Reply:
https://github.com/foundry-rs/starknet-foundry/pull/1851/commits/7f41085cdea12063307b7d8c854cbec2bbd7468e